### ISSUE TITLE
Schedule "tell tech your problems" session

### ISSUE THREAD

**Issue by strugee** [SeaGL/organization#334]
Going forward probably good to do this start-of-season and after Tech retrospective

**Comment by altsalt** [comment]
Good idea but needs to be backlogged for next year or changed to focus on a post-season session.
